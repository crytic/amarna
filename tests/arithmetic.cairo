func bar() -> (res):
    alloc_locals
    local x = 10 * 3618502788666131213697322783095070105623107215331596699973092056135872020481
    local x = 10 * 200 * 100 * 30
    local x = 10 * 200
    local y = 1/4
    return (res=x + y)
end