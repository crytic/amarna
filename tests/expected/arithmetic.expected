[arithmetic-add] in arithmetic.cairo:7:17
[arithmetic-div] in arithmetic.cairo:6:15
[arithmetic-mul] in arithmetic.cairo:3:15
[unused-function] in arithmetic.cairo:1:1